### PR TITLE
Fix support for React Portals

### PR DESCRIPTION
**Pull request by micburks**
Fixes https://github.com/fusionjs/fusionjs/issues/247

Short-term fix for portal support in client-side prepare

### CODE REVIEW

File: common/config/rush/yarn.lock [hunk 1]
```diff
@@ -1494,7 +1494,7 @@
 
 "@rush-temp/create-fusion-app@file:./projects/create-fusion-app.tgz":
   version "0.0.0"
-  resolved "file:./projects/create-fusion-app.tgz#3e4b87baaa08effcbc6e2c73442a5e2aa34b33cc"
```
Comment (mlmorg) [inline common/config/rush/yarn.lock]: What is this about?

Comment (micburks) [reply common/config/rush/yarn.lock]: After inspecting a little more, when I run `rush update` it removes the hashes, but when I run `rush update --full --purge` it adds the hashes.

@lhorie @rtsao What is expected here? Is this safe to ignore?

**Comment by ganemone** [comment]
I'm onboard with this. I think the next step is we do a breaking change that removes support for client side prepare all together.

**Comment by micburks** [comment]
!merge

**Pull request merged**